install/kubernetes/cilium/templates/cilium-nodeinit/serviceaccount.yaml

```
@@ -0,0 +1,11 @@
+{{- if .Values.nodeinit.enabled }}
```

This one should only be created if create is true and the account is enabled:

```suggestion
{{- if and .Values.nodeinit.enabled .Values.serviceAccounts.nodeinit.enabled  .Values.serviceAccounts.nodeinit.create }}
```